dev-util/gnome-builder/gnome-builder-44.2-r1.ebuild

```diff
@@ -272,7 +272,7 @@ pkg_postinst() {
 	readme.gentoo_print_elog
 
 	optfeature_header "Language support"
-	optfeature "Rust's Cargo build system" virtual/rust
+	optfeature "Rust's Cargo build system" dev-lang/rust
```

Comment:
Not right, needs to be dev-lang/rust dev-lang/rust-bin (see `man optfeature.eclass`).